portfolio/src/main/webapp/index.html

```diff
@@ -8,6 +8,7 @@
     
         <link rel="stylesheet" type="text/css" href="./css/indexStyle.css">
 
+        <script src="./script.js"/>
```

Comment:
nit: AFAIK we do not prefer relative paths in HTML. I believe you should change it to absolute path like `src="/script.js"`